src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4027,10 +4028,12 @@ private bool IsPossibleParameter(bool allowThisKeyword)
                 case SyntaxKind.OpenBracketToken: // attribute
                 case SyntaxKind.RefKeyword:
                 case SyntaxKind.OutKeyword:
-                case SyntaxKind.OpenParenToken:   // tuple
                 case SyntaxKind.ParamsKeyword:
                 case SyntaxKind.ArgListKeyword:
                     return true;
+                case SyntaxKind.OpenParenToken:   // tuple
+                    return IsFeatureEnabled(MessageID.IDS_FeatureTuples) || IsPredefinedType(this.CurrentToken.Kind);
```

Comment:
It is not obvious where the IsPredefinedType check came from. And, since we already know that this.CurrentToken.Kind is SyntaxKind.OpenParenToken, can it ever return true?